packages/image/src/index.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <div ref="container" class="el-image">
+  <div
+    ref="container"
+    class="el-image"
+    :class="$props.class"
```

Comment:
The newest code:
```
:class="rootClass"
:style="rootClass"
```
Because `class` is keyword of js, cannot use `:class="class"` directly, so I wrap it and `style` in setup.